ci-operator/config/openshift-assisted/assisted-events-scrap/openshift-assisted-assisted-events-scrap-master.yaml

```diff
@@ -0,0 +1,49 @@
+base_images:
+  assisted-service:
+    name: ocm-2.5
+    namespace: edge-infrastructure
+    tag: assisted-service
+  coreos_centos_stream8:
+    name: stream8
+    namespace: coreos
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: Dockerfile.build
+  from: coreos_centos_stream8
+  inputs:
+    assisted-service:
+      as:
+      - quay.io/ocpmetal/assisted-service:latest
```

Comment:
@lranjbar shouldn't the ``Dockerfile.build`` image use ``quay.io/edge-infrastructure/assisted-service:latest`` instead?
I don't remember if we're updating the old image anymore (or if we want to continue doing that)

Reply:
Anyway, now the Dockerfile uses quay.io/ocpmetal/assisted-service:latest ... so for now this is the right path

Reply:
The decision was made awhile back to move away from ocpmetal. This new Dockerfile should use edge-infrastructure and not ocpmetal.

Reply:
I vote 2 PRs and update it now so it is not forgotten about in a week or so.

Reply:
yeah let's do this